st2common/st2common/models/api/pack.py

```diff
@@ -120,5 +125,74 @@ def to_model(cls, config_schema):
         attributes = config_schema.attributes
 
         model = cls.model(pack=pack, attributes=attributes)
+        return model
 
+
+class ConfigAPI(BaseAPI):
+    model = ConfigDB
+    schema = {
+        "title": "Config",
+        "description": "Pack config.",
+        "type": "object",
+        "properties": {
+            "id": {
+                "description": "The unique identifier for the config.",
+                "type": "string"
+            },
+            "pack": {
+                "description": "The content pack this config belongs to.",
+                "type": "string"
+            },
+            "values": {
```

Comment:
Imo, `configs` would be a weird name. That's jut the attribute which holds the actual config values (since we obviously can't store those values as a top-level attributes on the model).